sig-wg-lifecycle.md

```diff
@@ -70,14 +83,19 @@ Sometimes it might be necessary to sunset a SIG or Working Group. SIGs/WGs may a
 ### Prerequisites for WG Retirement
 - [ ] Have completed the mission of the WG or have another reason as outlined in [wg-governance.md]
 
+### Prerequisites for UG Retirement
```

Comment:
I am guessing this is "any-of" rather than "all-of"

Reply:
I thought we weren't doing mailing lists for UGs?